firebase-firestore/src/test/java/com/google/firebase/firestore/model/FieldValueTest.java

```diff
@@ -137,7 +151,7 @@ public void testImplicitlyCreatesObjects() {
   @Test
   public void testCanOverwritePrimitivesWithObjects() {
     ObjectValue old = wrapObject("a", map("b", "old"));
-    ObjectValue mod = setField(old, "a", map("b", "mod"));
+    ObjectValue mod = setField(old, "a", wrapObject("b", "mod"));
```

Comment:
We should add an overload for `setField` that allows `map` to work inline so that we don't have think about this distinction.